internal/declarative/v2/reconciler.go

```diff
@@ -432,6 +433,7 @@ func (r *Reconciler) removeModuleCR(ctx context.Context, clnt Client, obj Object
 			if err := preDelete(ctx, clnt, r.Client, obj); err != nil {
 				r.Event(obj, "Warning", "PreDelete", err.Error())
 				// we do not set a status here since it will be deleting if timestamp is set.
```

Comment:
Is this comment still valid?

Reply:
yes because I don't set a new status, I only update the error with the current status